### DISCUSSION TITLE
Heroku deploy fails

### DISCUSSION THREAD

**Discussion by Louvivien** [solidusio/solidus#4501] [Troubleshooting]
**Solidus Version**:

**To Reproduce**
1. I go on the github and i click on the deploy app Heroku button
2. it fails

**Current behavior**

Here is the log: 

-----> Building on the Heroku-20 stack
-----> Determining which buildpack to use for this app
 !     Warning: Multiple default buildpacks reported the ability to handle this app. The first buildpack in the list below will be used.
			Detected buildpacks: Ruby,Node.js
			See https://devcenter.heroku.com/articles/buildpacks#buildpack-detect-order
-----> Ruby app detected
-----> Installing bundler 2.3.10
-----> Removing BUNDLED WITH version in the Gemfile.lock
-----> Compiling Ruby/Rails
-----> Using Ruby version: ruby-2.7.1
-----> Installing dependencies using bundler 2.3.10
       Running: BUNDLE_WITHOUT='development:test' BUNDLE_PATH=vendor/bundle BUNDLE_BIN=vendor/bundle/bin BUNDLE_DEPLOYMENT=1 bundle install -j4
       Your bundle only supports platforms ["x86_64-darwin-19"] but your local platform
       is x86_64-linux. Add the current platform to the lockfile with
       `bundle lock --add-platform x86_64-linux` and try again.
       Bundler Output: Your bundle only supports platforms ["x86_64-darwin-19"] but your local platform
       is x86_64-linux. Add the current platform to the lockfile with
       `bundle lock --add-platform x86_64-linux` and try again.
 !
 !     Failed to install gems via Bundler.
 !
 !     Push rejected, failed to compile Ruby app.
 !     Push failed

**Expected behavior**

Can you update the app to correct this?

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Louvivien** [comment]
I also tried to join Slack but it says I need to have an adress nebulab or supergood

**Reply by jarednorman** [reply]
There's an issue with our Slack setup. I'll look into that.

**Comment by jarednorman** [comment]
Sorry, I prematurely moved this to troubleshooting. This is a real bug, creating an issue from it.
